en/PlanView/pattern_corridor_scan.md

```diff
@@ -0,0 +1,96 @@
+# Corridor Scan (Plan Pattern)
+
```

Comment:
1. Is corridor scan supported on all vehicle types and flight stacks (it appears to be if you try all types but wanted to confirm).

Reply:
Yes